### PR TITLE
Fix the image resize on tree capture from being ignored

### PR DESCRIPTION
**Pull request by arunbakt**
The change utilizes the resizing of the image that happens when the captured image is really large.
The resized image data was ignored and this is fixed in this change.

### CODE REVIEW

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 1]
```diff
@@ -93,10 +93,14 @@ object ImageUtils {
         Timber.d("rotationAngle $rotationAngle")
 
         val matrix = Matrix()
-        matrix.setRotate(rotationAngle.toFloat(), bitmap.width.toFloat() / 2,
-                bitmap.height.toFloat() / 2)
-        return Bitmap.createBitmap(bitmap, 0, 0,
-                bmOptions.outWidth, bmOptions.outHeight, matrix, true)
+        matrix.setRotate(
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:96]: Ignore, ktlint formatting changes

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 2]
```diff
@@ -122,7 +126,7 @@ object ImageUtils {
     /**
      * out put image data pixels to console
      */
-     fun printImage(image: IntArray, rows: Int, cols: Int) {
+    fun printImage(image: IntArray, rows: Int, cols: Int) {
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:129]: ktlint format -ignore

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 3]
```diff
@@ -146,7 +150,10 @@ object ImageUtils {
         BitmapFactory.decodeFile(imagePath, bmOptions)
         val imageWidth = bmOptions.outWidth
 
-        var sampleSize = Math.ceil((imageWidth.toFloat() / scaleToWidth.toFloat()).toDouble()).toInt()
+        var sampleSize = Math.ceil(
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:153]: ktlint format -ignore

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 4]
```diff
@@ -190,7 +197,7 @@ object ImageUtils {
         val ran = kotlin.random.Random(System.currentTimeMillis()).nextInt(0, 10)
         println(ran)
         if (ran > 0) {
-           return applyGaussianKernel(image, ran, 2.0)
+            return applyGaussianKernel(image, ran, 2.0)
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:200]: ktlint format -ignore

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 5]
```diff
@@ -272,7 +279,11 @@ object ImageUtils {
      *  To test the focus detection, we need to blur some images.
      */
 
-    fun applyGaussianKernel(image: Array<Array<Int>>, kWidth: Int, sigma: Double): Array<Array<Int>> {
+    fun applyGaussianKernel(
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:282]: ktlint format -ignore

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 6]
```diff
@@ -366,15 +379,19 @@ object ImageUtils {
...
+            bitmap, 0, 0,
+            bmOptions.outWidth, bmOptions.outHeight, Matrix(), true
+        )
 
         val compressionQuality = 100
         val byteArrayBitmapStream = ByteArrayOutputStream()
-        rotatedBitmap.compress(Bitmap.CompressFormat.PNG, compressionQuality,
-            byteArrayBitmapStream)
-
-        return rotatedBitmap
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:377]: The return was ignored earlier by the calling code.

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 7]
```diff
@@ -413,14 +430,18 @@ object ImageUtils {
         /* Decode the JPEG file into a Bitmap */
         val bitmap = BitmapFactory.decodeFile(path, bmOptions)
 
-        val rotatedBitmap = Bitmap.createBitmap(bitmap, 0, 0,
-            bmOptions.outWidth, bmOptions.outHeight, Matrix(), true)
+        val rotatedBitmap = Bitmap.createBitmap(
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:433]: ignore -ktlint formatting

File: app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt [hunk 8]
```diff
@@ -413,14 +430,18 @@ object ImageUtils {
...
+            bitmap, 0, 0,
+            bmOptions.outWidth, bmOptions.outHeight, Matrix(), true
+        )
 
         val compressionQuality = 80
         val encodedImage: String
         val byteArrayBitmapStream = ByteArrayOutputStream()
-        rotatedBitmap.compress(Bitmap.CompressFormat.JPEG, compressionQuality,
-            byteArrayBitmapStream)
+        rotatedBitmap.compress(
```
Comment (arunbakt) [inline app/src/main/java/org/greenstand/android/TreeTracker/utilities/ImageUtils.kt:441]: Ignore - ktlint formatting

**Review by Elforama — APPROVED** [review]

**Pull request merged**